crypto/fipsmodule/evp/kem.c

```diff
@@ -0,0 +1,93 @@
+#include <openssl/base.h>
+#include <openssl/evp.h>
+#include <openssl/mem.h>
+
+#define SUCCESS (1)
```

Comment:
We just use `0` or `1` without macro's. Maybe sometime in the future.

Reply:
Would it be a problem to leave the macros? I personally find it more readable to use them, especially since the openssl convention of 1 for success and 0 for failure is different from what I'm used to (no data to back this up, but anecdotally it seems most *nix programs use 0 for success).